src/Microsoft.PowerShell.Security/security/CatalogCommands.cs

```diff
@@ -89,26 +89,35 @@ protected override void ProcessRecord()
 
             Collection<string> paths = new Collection<string>();
 
+            bool _ShouldProcess = false;
```

Comment:
Paul's suggestion is good, but I'm fine with a local variable as well - but the local variable should not use the convention for fields - it should be `shouldProcess`.